ogl.c

```diff
@@ -129,7 +130,8 @@ void ogl_init() {
 }
 
 void ogl_display() {
-	float fog_color[] = {0.5F,0.9098F,1.0F,1.0F};
+	printf("TEST\n");
```

Comment:
whoops

Reply:
Not going to pretend like I still remember what I did there, but after my experiences with the renderer, that one could totally have been on purpose.